releasenotes/notes/47740.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+releaseNotes:
+- |
+  **Updated** `istioctl experimental proxy-stauts` has been promoted to `istioctl proxy-status`. The old `istioctl proxy-status` command is removed.
```

Comment:
Should probably clarify the release note a little more.  Add something along the lines of `This promotion should not result in any loss of functionality.  Adds additional flags to support multi-control plane environments`